cloud/api-gateway/src/main/java/com/qpay/apigateway/exception/AuthenticationException.java

```diff
@@ -0,0 +1,11 @@
+package com.qpay.apigateway.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.FORBIDDEN)
+public class AuthenticationException extends RuntimeException {
+    public AuthenticationException() {
```

Comment:
I've already said in previous message to accept message from outside. When you hardcode it, it's not flexible. Don't you think that creating exception for every single case is too much